### DISCUSSION TITLE
Assign Task to Hours or Member

### DISCUSSION THREAD

**Discussion by dc143c** [solidtime-io/solidtime#291] [Q&A] [answered]
Hey, guys! I just deployed my self-hosted version of SolidTime and as a very big fan, I'm into using it to my freelances.
My main question is about the readme.md file which says:

> Tasks: Create and manage tasks and assign tasks to project members.

Even after a time working on this, I couldn't find where this feature is.
How do I assign tasks or manage task time? I really would love to track task time instead of generally doing time management with random labels.

**Comment by Onatcer** [comment, marked as answer]
Hi, sorry for the confusion. The text was just incorrect there, it should have been "assign tasks to projects". We do not currently have a feature that allows Tasks to be assigned. If this is something you would like to see, feel free to convert this Discussion into a feature request.
